app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  module V1
+    class BooksController < ApplicationController
+      include Wor::Paginate
+      rescue_from ActiveRecord::RecordNotFound, with: :error_404
+
+      def error_404
+        render json: [error: '404']
```

Comment:
This is really good, I like it but we have to make another iteration over this.
In Ruby, `[]` is an array, and in the other hand `{}` is a hash.
We ussualy render hashes for responses.

So first, I think we should change the array for one hash like this: `{ key: value }`

In this case, we tend to use `message` instead of `error`.
Then, the code of the error should not be inside the json, should be the status of the http response. In this case the status code is correct(404).
If you want to put it in outside of the json, you can do like this: `render json: { }, status: :not_found`
That will return an empty json with http response status `not_found`(404)

But we also want to return a message, so we can do something like:
`render json: { message: 'The record was not found' }, status: :not_found`